source/gameengine/Converter/BL_ActionActuator.cpp

```diff
@@ -170,6 +170,14 @@ bool BL_ActionActuator::Update(double curtime)
           if (useContinue) {
             obj->SetActionFrame(m_layer, m_localtime);
           }
+          else {
+            obj->SetPlayMode(m_layer, BL_Action::ACT_MODE_PLAY);
+            m_flag |= ACT_FLAG_PLAY_END;
+            // Swap the start and end frames
+            float temp = m_startframe;
+            m_startframe = m_endframe;
+            m_endframe = temp;
+          }
```

Comment:
We need to separate from LOOP_STOP and LOOP_END as we get a ping pong behaviour for loop stop/end this way.

Additionally, If we need a LOOP_PINGPONG case we need to add a new case instead of use continue.

This below modifications based in your code makes the work:
![image](https://user-images.githubusercontent.com/1015605/89147947-8e551380-d558-11ea-8464-07a5bcc9f051.png)

Reply:
Good work!!!